src/domain_tests/TestRunner.cs

```diff
@@ -888,6 +888,46 @@ import TestNamespace
     foo = sys.my_obj.Inner()
     print(foo)
     
+                    ",
+            },
+            new TestCase
+            {
+                // The C# code for this test doesn't matter.
```

Comment:
Please expand a little more: We're testing that the import hook in Python reloads properly, so the C# code for this test doesn't matter.

(in which case why have have any code at all? Can you compile an empty string?)